pypdns/__init__.py,sha256=PzDlH1ny616XH6_fK17VEiHuvMTBM_JWI3ZpOe3sHbY,134
pypdns/api.py,sha256=s9Lg4HEjUzTrsXufTsZBg66AswmoStq_VQaomHgx_g8,11746
pypdns/errors.py,sha256=z1MDSVuN0I553AczIVQhJA4U7JPoChkRAv1dOFaNLPw,474
pypdns/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pypdns-2.0.0.dist-info/LICENSE,sha256=o1p6jKkqoo0k2KXZ23F8zTPaRGIoF8Gu9ADxi-QiUyo,1576
pypdns-2.0.0.dist-info/WHEEL,sha256=DA86_h4QwwzGeRoz62o1svYt5kGEXpoUTuTtwzoTb30,83
pypdns-2.0.0.dist-info/METADATA,sha256=ABPzXOQULOsjXGs5ozohoFKF6ZpWNnWTMPSy-3tUwHg,1577
pypdns-2.0.0.dist-info/RECORD,,
